backend/src/openarchiefbeheer/utils/tests/e2e.py

```diff
@@ -49,10 +26,13 @@ async def browser_page_with_tracing(log_levels=["debug"]):
             and print(message.type.upper(), message),
         )
 
+        save_trace = True
```

Comment:
maybe make this conditional using an envvar (I think we can copy from open forms?)